integration_test/cases/repo.exs

```diff
@@ -152,6 +152,24 @@ defmodule Ecto.Integration.RepoTest do
     assert TestRepo.all(PostUserCompositePk) == []
   end
 
+  @tag :composite_pk
+  # TODO this needs a better name
+  test "insert, update and delete with associated composite pk #2" do
```

Comment:
I think we should remove the `associated` from the test before and then this one will be good. :)